src/doc/commands/commands_html.nit

```diff
@@ -48,7 +48,7 @@ redef class CmdEntity
 	redef fun to_html do
 		var mentity = self.mentity
 		if mentity == null then return ""
-		return mentity.html_link
+		return mentity.html_link.write_to_string
```

Comment:
why?

Reply:
All the other methods return directly printable HTML. With this fix, all `to_html` methods have the same behavior.